### PR TITLE
Due to the takeover of freenode we're moving to a different irc network.

### PR DESCRIPTION
**Pull request by abadger**
ansible channels have moved to libera.chat.

Do not merge until it's decided that we're moving to libera.chat (as opposed to oftc or another network): https://github.com/ansible-community/community-topics/issues/19

### CODE REVIEW

File: group-aws/README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ requests in the collection repositories.
 
 Planned work for both collections is tracked in the [AWS Project Board](https://github.com/orgs/ansible-collections/projects/4).
 ## Contact
-* [#ansible-aws IRC channel](https://webchat.freenode.net/?channels=ansible-aws) on Freenode.net
+* #ansible-aws IRC channel on [libera.chat](https://irc.libera.net/)
```
Comment (tadeboro) [inline group-aws/README.md:55]:
```suggestion
* #ansible-aws IRC channel on [libera.chat](https://irc.libera.chat/)
```

**Comment by abadger** [comment]
Thanks.  I messed up the links here too.  irc.libera.chat should be the
title and https://libera.chat should be the url

**Comment by gundalow** [comment]
This format of links works for me `ircs://irc.libera.chat/#ansible-community` as it seems like irccloud.com responds to `ircs://`

**Comment by abadger** [comment]
At today's irc meeting, it was decided to move to libera.chat.  This PR is now ready for review.

**Review by tadeboro — APPROVED** [review]

**Review by resmo — APPROVED** [review]

**Review by felixfontein — APPROVED** [review]

**Review by gundalow — COMMENTED** [review]
deleted, all good

**Review by gundalow — APPROVED** [review]

**Pull request merged**